File: FizzBuzz.cpp
//Program to print Fizz Buzz

#include <bits/stdc++.h>
using namespace std;

int main(void)
{
	int in;
	cout << "Enter a Number" << endl;
	cin >> in;
	for (int i=1; i<in; i++)
	{
		if (i%15 == 0)	
		printf ("FizzBuzz\t");
		else if ((i%3) == 0)
		printf("Fizz\t");		
		else if ((i%5) == 0)					
		printf("Buzz\t");				
		else		
		printf("%d\t", i);				
	}
	return 0;
}

File: FizzBuzz.java
// Java program to print Fizz Buzz
import java.io.BufferedReader;
import java.io.IOException;
import java.io.InputStreamReader;
public class FizzBuzz
{
	public static void main(String args[]) throws IOException
	{
        System.out.println("Enter a Number");
        BufferedReader reader = new BufferedReader(new InputStreamReader(System.in));
        int in = Integer.parseInt(reader.readLine());
        for (int i=1; i<in; i++)								
		{
			if (i%15==0)												
			System.out.print("FizzBuzz"+" ");
			else if (i%5==0)	
			System.out.print("Buzz"+" ");
			else if (i%3==0)	
			System.out.print("Fizz"+" ");				
			else
			System.out.print(i+" ");						
		}
	}
}
